app/routes/events/components/EventAdministrate/index.tsx

```diff
@@ -34,28 +36,40 @@ const EventAdministrateIndex = () => {
   const base = `/events/${eventId}/administrate`;
 
   return (
-    <Content>
-      <NavigationTab
-        title={event ? event.title : ''}
-        back={{
-          label: 'Tilbake',
-          path: '/events/' + event.slug,
-        }}
-        skeleton={fetching}
-      >
-        <NavigationLink to={`${base}/attendees`}>Påmeldinger</NavigationLink>
-        {event && canSeeAllergies(currentUser, event) && (
-          <NavigationLink to={`${base}/allergies`}>Allergier</NavigationLink>
-        )}
-        <NavigationLink to={`${base}/statistics`}>Statistikk</NavigationLink>
-        <NavigationLink to={`${base}/admin-register`}>
-          Adminregistrering
-        </NavigationLink>
-        <NavigationLink to={`${base}/abacard`}>Abacard</NavigationLink>
-      </NavigationTab>
-
+    <Page
+      title={`Administrerer: ${event.title}`}
+      back={
+        event?.slug && {
+          label: 'Tilbake til arrangement',
+          href: '/events/' + event.slug,
+        }
+      }
+      tabs={
+        <>
+          <NavigationTab href={`${base}/attendees`}>Påmeldinger</NavigationTab>
+          <Tooltip
+            disabled={canSeeAllergies(currentUser, event)}
```

Comment:
```suggestion
            disabled={!canSeeAllergies(currentUser, event)}
```
Shouldn't this only be disabled if you can't see allergies? Like the tab

Reply:
No the tooltip should only show when you can't see the allergies, to explain why you cannot see them. When allergies are visible, we don't really need to explain it.